Proposed repo template (#1)
hubmapconsortium/hubmap-template: README.md

# hubmap-template
Generic language-agnostic repo template with suggestions on best-practices:
As HuBMAP matures we will benefit from using standard software-engineering practices.
Because HuBMAP development is split across multiple repos, from multiple institutions,

> with a much of the work being done by grad students who will only be around for a short time,

Reviewer: ```suggestion
with some of the work being done by trainees who might only be around for a short time,
```

> establishing shared practices and structures is essential if the code base is to be maintained and developed over time.

Reviewer: ```suggestion
establishing shared practices and structures is particularly critical to maintain a high quality code base over time.
```

Once you've made a copy of this template, make a feature branch, and work through the TODOs.
If there's something you don't want to handle on the first pass, consider filing an issue for it. 

- [ ] **TODO**: Fill in the README sections below the `<hr>` at the bottom of this page.
- Github settings:
  - [ ] **TODO**: Under "General", turn off github features you don't need, and turn on others that aren't the default. Suggestions:
    - "Wikis" off.
    - "Projects" off.
    - "Allow merge commits" and "Allow rebase merging" off; "Allow squash merging" on.
    - "Automatically delete head branches" on.

>   - [ ] **TODO**: Under "Collborators and teams" add users besides yourself.

Reviewer: ```suggestion
  - [ ] **TODO**: Under "Collaborators and teams" add users besides yourself.
```
